backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,24 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler(object):
+    dice_values = list()
+    __re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def start_turn(self):
+        self.dice_values = dice.roll_many(constants.DEFAULT_DICE_COUNT)
+        self.__re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def re_roll_dice(self, indexes_of_dice_to_re_roll):
+        if self.__re_rolls_left > 0:
+            if type(indexes_of_dice_to_re_roll) == int:
+                self.dice_values[indexes_of_dice_to_re_roll] = dice.roll()
+
+            elif type(indexes_of_dice_to_re_roll) == list:
+                for val in indexes_of_dice_to_re_roll:
+                    self.dice_values[val] = dice.roll()
```

Comment:
For this, I would strongly suggest against allowing different types for the parameter. Instead, I would suggest only allowing a list of type integer. If there's only one, then they'll give a list with only one integer.

You may also want to consider making this two separate methods, one for doing a single roll with an integer and this method which allows a list.

Reply:
There's also no check here to make sure that what the client is sending makes sense. If they send an integer that's outside of the range of the dice, then it's going to throw an out of index error and explode.

Reply:
I don't think we need 2 methods. Just accept a list of ints and move on.
We do need error handling somewhere along the lines.